mibs/common/SNMPv2-SMI

```diff
@@ -0,0 +1,49 @@
+-- *****************************************************************
+-- SNMPv2-SMI.my:  SNMPv2 Structure of Management Information
+--
+-- May 1994, Jeffrey T. Johnson
+--
+-- Copyright (c) 1994,1996 by cisco Systems, Inc.
+-- All rights reserved.
```

Comment:
We don't appear to have the rights to share this file.

Reply:
I'll use the LibraNMS variation of this then. It was extracted from the RFC without this copyright.

It's questionable if Cisco can even copyright something that's already copyright of the IETF.

Reply:
So what's the license on that?